landing/js/welcome.js

```diff
@@ -18,6 +18,9 @@
         scrollTo: {
           behavior: 'smooth',
           block: 'center'
+        },
+        popperOptions: {
+          modifiers: [{ name: "offset", options: { offset: [0, 32] } }]
```

Comment:
Do we want this offset on the landing or was it leftover from where you were playing with it?

Reply:
I think it's nice to have these examples in while developing, but I can remove if you prefer.

Reply:
Does this not change the style of the landing page though? I think we would want to keep the landing page styled as it was, right?